signal/cosmos.go

```diff
@@ -78,7 +99,7 @@ func (c *Cosmos) setTrack(config config.Config) error {
 		return fmt.Errorf("%s report is nil, bailing out.", c.Name)
 	}
 
-	log.Warnf("REPORT:\n%+v", c.Report)
+	log.Infof("Installed cosmos packages: %s", c.Report.String())
```

Comment:
You shouldn't neat to use `c.Report.String()` if you're formatting with `%s`. `c.Report` should work the same.